Dart/testSrc/com/jetbrains/lang/dart/typing/DartTypingTest.java

```diff
@@ -607,4 +607,11 @@ public void testEnterAfterEqualsProperty() {
                  "o.x =\n" +
                  "    <caret>");
   }
+
+  public void testEnterBeforeStringParam() {
```

Comment:
This test passes without any other code changes. I guess it's because `return` in test data is not inside a function.